### ISSUE TITLE
`search` Infuse's version is wrong

### ISSUE THREAD

**Issue by Fei-Wang** [mas-cli/mas#461]
## Your Environment

- `mas version`: 1.8.6
- macOS version: 12.3.1

### mas Install Method

- [x] `brew install mas` (homebrew-core)
- [] [mas-cli/tap](https://github.com/mas-cli/homebrew-tap)
- [] `.pkg` installer from [releases](https://github.com/mas-cli/mas/releases)
- [] Built from source
  - Fork/branch: ? (e.g. mas-cli/main)
  - Xcode version: 10.?

## Describe the Bug

> mas search infuse's version is 7.3.9, it is wrong. 

## To Reproduce
```mas search infuse```

## Screenshots, Terminal Output
<img width="429" alt="image" src="https://user-images.githubusercontent.com/11441526/162870995-300eec12-4514-474d-919a-8209f469ff0c.png">

**Comment by virajp** [comment]
Similar issue I am facing with "VPN Unlimited" which is actually version 8.6.9 (as per App Store) but mas-cli is taking as 9.1.8 (not sure from where).

❯ mas outdated
694633015 VPN Unlimited® (8.6.9 -> 9.1.8)

**Comment by chasetb** [comment]
Same with "1Blocker" on two different computers: MacBook Pro (16-inch, 2019) and Mac Studio both running macOS 13.1 and mas version `1.8.6`
```
❯ mas info 1365531024
1Blocker - Ad Blocker 5.3.3 [0.0]
By: 1Blocker, LLC
Released: 2022-12-23
Minimum OS: 14.2
Size: 76.9 MB
From: https://apps.apple.com/us/app/1blocker-ad-blocker/id1365531024?uo=4
```

The App Store shows the latest version is `4.1.10` and published on 2022-12-16.

**Comment by chasetb** [comment]
> ```
> ❯ mas info 1365531024
> 1Blocker - Ad Blocker 5.3.3 [0.0]
> By: 1Blocker, LLC
> Released: 2022-12-23
> Minimum OS: 14.2
> Size: 76.9 MB
> From: https://apps.apple.com/us/app/1blocker-ad-blocker/id1365531024?uo=4
> ```

I wonder if the "Minimum OS" is somewhat to blame here. The latest version of macOS right now is 13.1 but the info for 1Blocker shows the Minimum OS to be 14.2. Compare that to the current info for Numbers, for example, where the Minimum OS is 12.0: 
```
❯ mas info 409203825
Numbers 12.2.1 [0.0]
By: Apple Inc.
Released: 2022-11-29
Minimum OS: 12.0
Size: 253.9 MB
From: https://apps.apple.com/us/app/numbers/id409203825?mt=12&uo=4
```
Now look at the info for Infuse, which puts the Minimum OS at 14.0:
```
❯ mas info 1136220934
Infuse • Video Player 7.4.10 [0.0]
By: Firecore, LLC
Released: 2022-12-26
Minimum OS: 14.0
Size: 104 MB
From: https://apps.apple.com/us/app/infuse-video-player/id1136220934?uo=4
```

**Comment by rgoldberg** [comment]
Closing as a duplicate of #491. PR #496 fixed the issue. It will be released in the upcoming version 1.8.7, which is currently in beta.

If you think this issue is different than #491, please let us know.
